src/main/webapp/index.html

```diff
@@ -5,40 +5,66 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  
     <title>GPostCard</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/env.css">
+    <link rel="stylesheet" href="style/style.css">
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBFO8JyMu6f5bvQQG3c9KfEBayfbISs1gc"></script>
 
   </head>
   <body onload="login()" id="index_body">
+
+    <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="main-nav">
+      <a class="navbar-brand" href="index.html"> <img src="image/logo.png" class="navbar-logo" alt=""></a>
```

Comment:
for accessibility reasons, always provide alternative contents for multimedia.
For images that means use of meaningful alternative text (alt) and for video and audio transcripts and captions, if available.